templates/dt_template/README.md

```diff
@@ -0,0 +1,80 @@
+# Deployed Topology 
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change                | Date             |
```

Comment:
Thinking more about this, do we really need this table? We use git, we can trace the history and I fear this table will become quickly outdated.